l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py

```diff
@@ -8,13 +8,25 @@
 
 class TestL10nNlXafAuditfileExport(TransactionCase):
 
+    def setUp(self):
+        super(TestL10nNlXafAuditfileExport, self).setUp()
+        self.fiscalyear = self.env['account.fiscalyear'].browse(
+            self.env['account.fiscalyear'].find(exception=False) or []
```

Comment:
You seem to cover the case in which there is no current fiscal year. Does that not break the tests below?

Reply:
no. What we need to do here is to take the current fiscal year because some other module creates an old fiscal year with no invoices

Reply:
So the empty list is a valid fallback?

Reply:
ah, that's what you mean. No, that's just a habit

Reply:
Do you want to leave it in? As you can see, it confuses the more simpler minds like me.

Reply:
I changed the whole thing to also not decrease coverage